File: Examples/CSharp/WorkingWithResources/GetResourceCosts.cs
namespace Aspose.Tasks.Examples.CSharp.WorkingWithResources
{
    using System;

    internal class GetResourceCosts
    {
        public static void Run()
        {
            var dataDir = RunExamples.GetDataDir(System.Reflection.MethodBase.GetCurrentMethod());

            //ExStart:GetResourceCosts
            //ExFor: Rsc.Cost
            //ExFor: Rsc.ACWP
            //ExFor: Rsc.BCWS
            //ExFor: Rsc.BCWP
            //ExSummary: Shows how to read resource costs.
            var project = new Project(dataDir + "ResourceCosts.mpp");

            // Display all resources costs
            foreach (var res in project.Resources)
            {
                if (res.Get(Rsc.Name) == null)
                {
                    continue;
                }

                Console.WriteLine(res.Get(Rsc.Cost));
                Console.WriteLine(res.Get(Rsc.ACWP));
                Console.WriteLine(res.Get(Rsc.BCWS));
                Console.WriteLine(res.Get(Rsc.BCWP));
            }
            //ExEnd:GetResourceCosts
        }
    }
}

File: Examples/VisualBasic/WorkingWithResources/GetResourceCosts.vb

Namespace WorkingWithResources
    Public Class GetResourceCosts
        Public Shared Sub Run()
            ' The path to the documents directory.
            Dim dataDir As String = RunExamples.GetDataDir(System.Reflection.MethodBase.GetCurrentMethod().DeclaringType.FullName)

            ' ExStart:GetResourceCosts
            ' Create project instance
            Dim project1 As New Project(dataDir + "ResourceCosts.mpp")

            ' Display all resources costs
            For Each res As Resource In project1.Resources
                If res.[Get](Rsc.Name) IsNot Nothing Then
                    Console.WriteLine(res.[Get](Rsc.Cost))
                    Console.WriteLine(res.[Get](Rsc.ACWP))
                    Console.WriteLine(res.[Get](Rsc.BCWS))
                    Console.WriteLine(res.[Get](Rsc.BCWP))
                End If
            Next
            ' ExEnd:GetResourceCosts
        End Sub
    End Class
End Namespace

